obspy/clients/fdsn/mass_downloader/utils.py

```diff
@@ -542,9 +542,8 @@ def get_mseed_filename(str_or_fct, network, station, location, channel,
     if callable(str_or_fct):
         path = str_or_fct(network, station, location, channel, starttime,
                           endtime)
-    elif ("{network}" in str_or_fct) and ("{station}" in str_or_fct) and \
-            ("{location}" in str_or_fct) and ("{channel}" in str_or_fct) and \
-            ("{starttime}" in str_or_fct) and ("{endtime}" in str_or_fct):
+    elif any("{%s}" % key in str_or_fct for key in ("network", "station", \
+                "location", "channel", "starttime", "endtime")):
```

Comment:
please get rid of redundant backslash (inside brackets) and fix indent of continuation line, see circle CI with code check results